### PR TITLE
add base context to the appendix with a corresponding hash

### PR DESCRIPTION
**Pull request by yancyribbens**
In this PR I address issue https://github.com/w3c/vc-data-model/issues/549 by adding the flattened base context with sorted keys to appendix B.  A `SHA256` hash of the flattened sorted base context is also added to the appendix.

A follow-up requires updating the URI `https://www.w3.org/2018/credentials/v1` with the flattened context from this PR.  The uploaded context should be verified to make sure the resulting SHA256 hash matches.

***
<a href="https://pr-preview.s3.amazonaws.com/yancyribbens/vc-data-model/pull/587.html" title="Last updated on May 26, 2019, 8:33 AM UTC (a1de4a8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/587/446c421...yancyribbens:a1de4a8.html" title="Last updated on May 26, 2019, 8:33 AM UTC (a1de4a8)">Diff</a>

### CODE REVIEW

**Comment by yancyribbens** [comment]
@dlongley was recommended to help review.

File: index.html [hunk 1]
```diff
@@ -1024,7 +1024,8 @@ <h3>Contexts</h3>
           <dd>
 The value of the <code>@context</code> <a>property</a> MUST be an ordered set
 where the first item is a <a>URI</a> with the value
-<code>https://www.w3.org/2018/credentials/v1</code>. Subsequent items in the
+<code>https://www.w3.org/2018/credentials/v1</code>. A copy of the base context is
+provided for refrence in the appendix <a href="#base-context"></a>. Subsequent items in the
 array MUST be composed of any combination of <a>URIs</a> or objects that
 express context information. It is RECOMMENDED that each <a>URI</a> in the
```
Comment (TallTed) [inline index.html]:
```
provided for refrence in the appendix <a href="#base-context"></a>. Subsequent items in the
array MUST be composed of any combination of <a>URIs</a> or objects that
express context information.
```
->
```
provided for reference in the appendix <a href="#base-context"></a>. Subsequent items in the
array MUST be composed of a combination of <a>URIs</a> and/or objects that
expresses context information.
```
* `refrence` -> `reference`
* `any combination` -> `a combination`
* `or` -> `and/or`
* `express` -> `expresses`

File: index.html [hunk 2]
```diff
@@ -4861,6 +4862,122 @@ <h3>Fitness for Purpose</h3>
       </section>
     </section>
 
+  </section>
+
+  <section class="appendix informative">
+    <h2>Base Context</h2>
+      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in his section for convience.
```
Comment (TallTed) [inline index.html]:
```      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in his section for convience.
```
->
```
The base context, located at <code>https://www.w3.org/2018/credentials/v1</code> 
with SHA-256 digest 
<strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong>,
can be used to implement a local cached copy.  The aforementioned context is 
provided in this section for convenience.
```
* added line breaks
* tweaked styling around and naming of `SHA-256` (properly requires hyphen)
* `convience` and `his` spelling -> `convenience` and `this`

File: index.html (hunk 1 above)
Comment (yancyribbens) [reply index.html]: The typo I introduced was fixed, although the next 3 bullets where issues present prior to my PR.  While I don't disagree, those changes should be addressed separately I think (unless I missed something).

Comment (TallTed) [reply index.html]: Fair enough. (Better not to mark conversation as resolved if there's a pending action, such as my new PR.)

File: index.html [hunk 3]
```diff
@@ -4866,7 +4866,7 @@ <h3>Fitness for Purpose</h3>
 
   <section class="appendix informative">
     <h2>Base Context</h2>
-      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in this section for convenience.
+      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA-256 digest <strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong> can be used to implement a local cached copy.  The aforementioned context is provided in this section for convenience.
```
Comment (TallTed) [inline index.html]: github won't let me make a PR against your fork, so here's my suggested edit, as a multi-line (fitting existing doc markup style) codeblock you can copy and paste, replacing your single-line 4869 --
```
      <p>
The base context, located at 
<code>https://www.w3.org/2018/credentials/v1</code> with SHA-256 digest 
<strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong>, 
can be used to implement a local cached copy.  For convenient reference,
the content of that file is also provided below.
      </p>
```

**Comment by dlongley** [comment]
@yancyribbens, I've got some changes to the context that I'll try to create a PR for when I get a chance -- against your fork.

**Comment by msporny** [comment]
Can you please merge https://github.com/yancyribbens/vc-data-model/pull/1 , @yancyribbens? Once you do that, we can pull this in and close the issue.

**Comment by yancyribbens** [comment]
@msporny @dlongley I've pulled in the changes and updated the `sha256` hash as well as the context referenced in `index.html`.  I think this is ready now.

**Pull request merged**